src/app/components/itemlist.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import useFetchAdvocates, { Advocate } from "../hooks/useFetchAdvocates";
+
+interface ItemListProps {
+	searchText: string | null;
+}
+
+const ItemList: React.FC<ItemListProps> = ({ searchText }: ItemListProps) => {
+
+	const [currentPage, setCurrentPage] = useState(1);
+	const [pageSize,] = useState(10);
```

Comment:
If we wanted to allow an option for the user to select a pageSize they prefer. We can just add `setPageSize` and implement a dropdown component to accommodate this feature.